00_Intro/bedrock_boto3_setup.ipynb

```diff
@@ -137,6 +141,7 @@
     "    assumed_role=os.environ.get(\"BEDROCK_ASSUME_ROLE\", None),\n",
     "    endpoint_url=os.environ.get(\"BEDROCK_ENDPOINT_URL\", None),\n",
     "    region=os.environ.get(\"AWS_DEFAULT_REGION\", None),\n",
+    "    mode='control-plane'\n",
```

Comment:
please remove control-plane and data-plane references.

Reply:
Updated